src/helm/benchmark/run.py

```diff
@@ -115,7 +115,7 @@ def add_run_args(parser: argparse.ArgumentParser):
     parser.add_argument(
         "-o", "--output-path", type=str, help="Where to save all the output", default="benchmark_output"
     )
-    parser.add_argument("-n", "--num-threads", type=int, help="Max number of threads to make requests", default=4)
+    parser.add_argument("-n", "--num-threads", type=int, help="Max number of threads to make requests", default=1)
```

Comment:
Why change to 1?

Reply:
This was just for debugging purposes. I wanted to be able to step through an inference request in pdb so I could find a mysterious error. Thus, I needed to only have the single python process (parallelism == 1 [here](https://github.com/stanford-crfm/helm/blob/80ecb204a9fed6a54a53e1f5950e9c1e145acddc/src/helm/common/general.py#L204)) instead of creating a pool of 4 processes. I've reverted this.